users.go

```diff
@@ -305,15 +305,17 @@ func (user *User) Mute() error {
 	insta := user.inst
 	data, err := insta.prepareData(
 		map[string]interface{}{
```

Comment:
Can you add an option to select 'story', 'feed', 'all' for it?
Same as https://github.com/mgp25/Instagram-API/blob/0a9d2b911203c55e3262bab0b8e9466feba7d5e0/src/Request/People.php#L1005

Reply:
I've added a new commit which covers this and removes code duplication.